<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Controls</class>
 <widget class="QWidget" name="Controls">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>150</width>
    <height>393</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Function</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="sineRadio">
     <property name="text">
      <string>Sine</string>
     </property>
     <property name="checked">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="triangleRadio">
     <property name="text">
      <string>Triangle</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="squareRadio">
     <property name="text">
      <string>Square</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="noiseRadio">
     <property name="text">
      <string>Noise</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="oneDivSineRadio">
     <property name="text">
      <string>1 / sin(x)</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QRadioButton" name="sineOneDivRadio">
     <property name="text">
      <string>sin(1 / x)</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>17</width>
       <height>13</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="runButton">
     <property name="text">
      <string>Run</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_3">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>17</width>
       <height>13</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QPushButton" name="add1kButton">
     <property name="text">
      <string>Add 1000 points</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="add10kButton">
     <property name="text">
      <string>Add 10k points</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QPushButton" name="add100kButton">
     <property name="text">
      <string>Add 100k points</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>37</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
